hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -225,7 +231,7 @@ public String getCertificate(String certSerialId) throws IOException {
         certSerialId);
     try {
       X509Certificate certificate =
-          certificateServer.getCertificate(certSerialId);
+          rootCertificateServer.getCertificate(certSerialId);
```

Comment:
No, as scmcertserver runs on all SCM's used the scmcertserver

Underlying both CertificateServers use the same certstore. So, the semantics of this API is not changed, it can get any certificate.